defaults/main/install.yml

```
@@ -17,12 +17,19 @@ apache_conf_dir: "{{ apache_etc_dir }}/conf.d"
 apache_log_dir: "/var/log/{{ apache_service_name }}"
 
 rpm_repo_url: "https://yum.osc.edu/ondemand/2.0/ondemand-release-web-2.0-1.noarch.rpm"
+apt_repo_url: "https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest_2_all.deb"
+
 rpm_repo_key: "https://yum.osc.edu/ondemand/RPM-GPG-KEY-ondemand"
+deb_repo_key: "https://yum.osc.edu/ondemand/DEB-GPG-KEY-ondemand"
```

It doesn't make a difference but in the future it might
```suggestion
deb_repo_key: "https://apt.osc.edu/ondemand/DEB-GPG-KEY-ondemand"
```